src/swapd/runtime.rs

```diff
@@ -1002,6 +1044,18 @@ impl Runtime {
                     Some(remote_commit),
                 );
 
+                std::thread::sleep(Duration::from_secs_f32(10.0));
+                let btc_fee_task = self.syncer_state.estimate_fee_btc();
+                endpoints.send_to(
+                    ServiceBus::Ctl,
+                    self.identity(),
+                    self.syncer_state.bitcoin_syncer(),
+                    Request::SyncerTask(btc_fee_task),
+                )?;
+
+                // syncer takes too long to give a fee
+                std::thread::sleep(Duration::from_secs_f32(10.0));
```

Comment:
I've run the PR successfully for 10 times with 0.5 seconds delay instead here. I think it is safe enough to at least halve it.

Reply:
The `swap_bob_maker_user_abort_sweep_btc` functional test aborts right after funding, so it is perfect to test this fast.